webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -59,26 +59,30 @@
                     <optgroup label=" ">
                         <option value="Current Versions">Current Versions</option>
                     </optgroup>
+
+                    {% set pvs = filter_featured_versions(active_versions[product]) %}
+                    {% if pvs %}
                     <optgroup label=" ">
-                    {% for pv in active_versions[product] %}
-                        {% if pv.is_featured %}
+                    {% for pv in pvs %}
                         <option
                             value="{{ pv.version }}"
                             {% if pv.version == version %}selected{% endif %}
                         >{{ pv.version }}</option>
-                        {% endif %}
-                    {% endfor  %}
+                    {% endfor %}
                     </optgroup>
+                    {% endif %}
+
+                    {% set pvs = filter_not_featured_versions(active_versions[product]) %}
+                    {% if pvs %}
                     <optgroup label=" ">
-                    {% for pv in active_versions[product] %}
-                        {% if not pv.is_featured %}
+                    {% for pv in pvs %}
                         <option
                             value="{{ pv.version }}"
                             {% if pv.version == version %}selected{% endif %}
                         >{{ pv.version }}</option>
-                        {% endif %}
-                    {% endfor  %}
+                    {% endfor %}
                     </optgroup>
+                    {% endif %}
```

Comment:
All of these changes were necessary so that you don't get this:
<img width="349" alt="screen shot 2017-03-31 at 9 56 27 am" src="https://cloud.githubusercontent.com/assets/26739/24553482/56d8442c-15f8-11e7-964e-bcc34dcd7021.png">